builder/osc/common/step_update_omi_test.go

```diff
@@ -0,0 +1,41 @@
+package common
+
+import (
+	"testing"
+
+	"github.com/hashicorp/packer-plugin-sdk/multistep"
+
+	"bytes"
+	"context"
+
+	packersdk "github.com/hashicorp/packer-plugin-sdk/packer"
+)
+
+// Create statebag for running test
+func tState() multistep.StateBag {
+	state := new(multistep.BasicStateBag)
+	accessConfig := &AccessConfig{}
+
+	state.Put("ui", &packersdk.BasicUi{
+		Reader: new(bytes.Buffer),
+		Writer: new(bytes.Buffer),
+	})
+	state.Put("omis", map[string]string{"us-west-2": "omi-12345"})
+	state.Put("snapshots", map[string][]string{"us-west-2": {"snap-0012345"}})
+	state.Put("accessConfig", accessConfig)
+	return state
+}
+
+func TestUpdateOmi(t *testing.T) {
+
+	StepUpdateOMIAttributes := StepUpdateOMIAttributes{
+		AccountIds:         []string{},
+		SnapshotAccountIds: []string{},
+		RawRegion:          "us-west-2",
+		GlobalPermission:   true,
+	}
+	state := tState()
+
+	StepUpdateOMIAttributes.Run(context.Background(), state)
```

Comment:
What does the test do ?

Reply:
yes, doesn't this test return an error if something (the StepUpdateOMIAttributes) doesn't work ?